exporter/awskinesisexporter/internal/batch/encode_jaeger.go

```diff
@@ -12,47 +12,49 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package translate
+package batch
 
 import (
-	awskinesis "github.com/signalfx/opencensus-go-exporter-kinesis"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/model/pdata"
 
 	jaegertranslator "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/jaeger"
 )
 
-type jaeger struct {
-	kinesis *awskinesis.Exporter
-}
+type jaeger struct{}
```

Comment:
Need to update to store max allowed batch size and record size when creating a batch to forward to kinesis.